### PR TITLE
1-list-all-hardware

### PR DESCRIPTION
**Pull request by kjacoby3**
This PR creates the API service that retrieves all hardware from hardware table on database/

[Issue Link](https://github.com/YCPHacks/hardware-management-api/issues/1)

### CODE REVIEW

File: routes/hardware/index.js
Comment (Trey-Mac) [inline routes/hardware/index.js:1]: Good work Paul

**Review by Estrawser — DISMISSED** [review]

**Review by pwalter1014 — APPROVED** [review]
Good job! Nice addition of name column.

File: routes/hardware/index.js [hunk 1]
```diff
@@ -1,9 +1,37 @@
 'use strict'
+const mysqlx = require('@mysql/xdevapi');
 
 module.exports = async function (fastify, options) {
   fastify.get('/', async function (request, reply) {
-    reply.code(200);
+    //reply.code(200);
```
Comment (pwalter1014) [inline routes/hardware/index.js]: Remove comments

**Review by pwalter1014 — CHANGES_REQUESTED** [review]

**Review by pwalter1014 — APPROVED** [review]
Thank you

**Review by Estrawser — APPROVED** [review]
Looks good

**Review by Trey-Mac — APPROVED** [review]
this has been approved

**Pull request merged**